From 81d22f808edbc3de23cba9afa8aca3328d6e0ac4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=82=B1=E5=BB=BA=E6=96=8C?= <qjb@rock-chips.com>
Date: Wed, 26 Dec 2012 17:34:43 +0800
Subject: [PATCH] rt5631: Fix compilation warning

sound/soc/codecs/rt5631.c: In function 'rt5631_probe':
sound/soc/codecs/rt5631.c:2008: error: assignment from incompatible pointer type
---
 sound/soc/codecs/rt5631.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/rt5631.c b/sound/soc/codecs/rt5631.c
index e1d59b76ede9..2536f86e3cbe 100755
--- a/sound/soc/codecs/rt5631.c
+++ b/sound/soc/codecs/rt5631.c
@@ -277,7 +277,7 @@ static int rt5631_reg_init(struct snd_soc_codec *codec)
 	return 0;
 }
 //bard 7-16 s
-void rt5631_adc_on(void)
+void rt5631_adc_on(struct work_struct *work)
 {
 	int val;
 
-- 
2.35.3

